lib/c-api/Cargo.toml

```diff
@@ -93,4 +93,4 @@ cranelift-backend = ["cranelift"]
 llvm-backend = ["llvm"]
 
 [build-dependencies]
-cbindgen = "0.15"
+cbindgen = "0.18"
```

Comment:
Please don't update `cbindgen` along with other deps. It's a big change that we must monitor carefully :-).

Reply:
Nevervind, I'm reviewing the changes now.

Reply:
Following up on this, because we  ended up reverting back to structopt having things unrelated to this PR in the diff does make it a bit harder to read